src/include/wiredtiger.h.in

```diff
@@ -5779,7 +5779,7 @@ struct __wt_key_provider {
      *
      * @param kp the WT_KEY_PROVIDER instance.
      * @param session the current WiredTiger session.
-     * @param keysp the WT_CRYPT_KEYS to hold the new key (if changed).
+     * @param[out] keysp the WT_CRYPT_KEYS to hold the new key (if changed).
```

Comment:
The @param is used by doxygen to generate docs. I think doxygen will fail to generate this properly. Worth double checking